packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts

```diff
@@ -123,6 +123,10 @@ export interface IBasePickerProps<T> extends React.Props<any> {
    * A callback used to modify the input string.
    */
   onInputChange?: (input: string) => string;
+  /**
+   * A callback to override the default behavior of adding the selected suggestion on dismiss.
+   */
+  onDismiss?: (selectedItem?: T, ev?: any) => void;
```

Comment:
last little nit. our code convention is to have the event first, and other props after.